deploy/ui/form.json

```diff
@@ -1122,6 +1122,47 @@
                           "displayWhen": "external",
                           "visible": false,
                           "fields": [
+                             {
+                              "label": "Enable Extension image stream",
+                              "type": "checkbox",
+                              "required": false,
+                              "description": "When set to true will configure the KIE Server with Extension Image Stream.",
+                              "jsonPath": "$.spec.objects.servers[*].build",
+                              "originalJsonPath": "$.spec.objects.servers[*].build",
+                              "fields": [
+                                  {
+                                  "label": "Enable Extension image stream",
+                                  "type": "fieldGroup",
+                                  "jsonPath": "$.spec.objects.servers[*].build",
+                                  "originalJsonPath": "$.spec.objects.servers[*].build",
+                                  "displayWhen": "true",
+                                  "visible": false,
+                                  "fields": [
+                                    {
+                                      "label": "Extension image stream tag",
+                                      "type": "text",
+                                      "required": true,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTag",
+                                      "description": "ImageStreamTag definition for the image containing the drivers and configuration. For example, custom-driver-image:7.7.0."
+                                    },
+                                    {
+                                      "label": "Extension image stream tag namespace",
+                                      "type": "text",
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageStreamTagNamespace",
+                                      "description": "Namespace within which the ImageStream definition for the image containing the drivers and configuration is located. Defaults to openshift namespace."
+                                    },
+                                    {
+                                      "label": "Extension image install Dir",
+                                      "type": "text",
+                                      "required": false,
+                                      "jsonPath": "$.spec.objects.servers[*].build.extensionImageInstallDir",
+                                      "description": "scription: Full path to the directory within the extensions image where the extensions are located (e.g. install.sh, modules/, etc. Do not change this field unless you have sure about the changes that you is doing)."
```

Comment:
```suggestion
                                      "description": "Full path to the directory within the extensions image where the extensions are located (e.g. install.sh, modules/, etc.)."
```

Reply:
I will do the changes, Do you want  the changes in CRD too (https://github.com/kiegroup/kie-cloud-operator/blob/master/deploy/crds/kieapp.crd.yaml#L253)

Reply:
please do.

Reply:
Done.